HW1/task1.2/index.js

```diff
@@ -0,0 +1,14 @@
+import fs from 'fs';
+const csvtojson = require("csvtojson");
+
+const writeStream = fs.createWriteStream("./bookList.txt", 'UTF-8');
+
+
+csvtojson().fromFile('./csv/bookList.csv').then((data) => {
```

Comment:
fromFile loads everything in RAM. According to evaluation criteria that's only 3 out of 5 points.

4. Task 1.2 is fulfilled to the full extent;the file is not loaded fully in the RAM(pipeline method https://nodejs.org/api/stream.html#stream_stream_pipeline_streams_callback).

https://www.npmjs.com/package/csvtojson#use-stream